design/cassandra/cluster-creation.md

```diff
@@ -0,0 +1,242 @@
+          command:
+            - "/bin/bash"
+            - "-c"
+            - |
+              set -e
+              CASSANDRA_CONFIG="/etc/cassandra"
+              # first arg is `-f` or `--some-option`
+              # or there are no args
+              if [ "$#" -eq 0 ] || [ "${1#-}" != "$1" ]; then
+              	set -- cassandra -f "$@"
+              fi
+              # allow the container to be started with `--user`
+              if [ "$1" = 'cassandra' -a "$(id -u)" = '0' ]; then
+              	chown -R cassandra /var/lib/cassandra /var/log/cassandra "$CASSANDRA_CONFIG"
+              	exec gosu cassandra "$BASH_SOURCE" "$@"
+              fi
+              mkdir -p /mnt/shared/default
+              # Copy default config files
+              cp -a "$CASSANDRA_CONFIG"/* /mnt/shared/default
+              # Get custom config files
+              while ! [ -d "$SHARED_VOLUME"/custom ]; do sleep 10; done; cp -a "$SHARED_VOLUME"/default/* "$CASSANDRA_CONFIG"
+              # TODO: get plugins (jolokia, jmx-exporter)
+              # Start cassandra
+              exec "$@"
+          ports:
+            - containerPort: 7000
+              name: intra-node
+            - containerPort: 7001 
+              name: tls-intra-node
+            - containerPort: 7199 
+              name: jmx
+            - containerPort: 9042
+              name: cql
+            - containerPort: 9160
+              name: thrift 
+          env:
+            - name: SHARED_VOLUME
+              value: /mnt/shared
+            - name: POD_IP
+              valueFrom:
+              fieldRef:
+                fieldPath: status.podIP
+          resources:
+            limits: 
+              cpu: <cassandra-cpu>
+              memory: <cassandra-mem>
+            requests: 
+              cpu: <cassandra-cpu>
+              memory: <cassandra-mem>
+          volumeMounts: 
+            - name: <sts-name>-data
+              mountPath: /var/lib/cassandra
+            - name: shared
+              mountPath: /mnt/shared
+              readOnly: true
+        - name: sidecar
+          image: rook/rook
+          imagePullPolicy: Always
+          command:
+          - "/tini"
+          - "--"
+          - "/usr/local/bin/rook"
+          env:
+            - name: SHARED_VOLUME
+              value: /mnt/shared
+          resources:
+            limits:
+              cpu: <sidecar-cpu>
+              memory: <sidecar-mem>
+            requests:
+              cpu: <sidecar-cpu>
+              memory: <sidecar-mem>
+          volumeMounts:
+           - name: shared
+             mountPath: /mnt/shared
+  volumeClaimTemplates:
+    - metadata:
+        name: <sts-name>-data
+      spec:
+        storageClassName: <sc-name>
+        accessModes: [ "ReadWriteOnce" ]
+        resources: 
+          requests:
+            storage: <storage-request>
+```
+
+
+## Appendix B 
+
+The options that are of interest to the operator are:
+
+### cassandra.yaml
```

Comment:
is the 1st column in this table standard cassandra config values?

Reply:
The first column is the name of the option in the `cassandra.yaml` config file, which is the main configuration file for cassandra.